src/bindist/autodos7.bat

```diff
@@ -0,0 +1,6 @@
+@ECHO Off
+PATH D:\OPENDOS;D:\EMUBIN;D:\
+VERIFY OFF
+PROMPT [OpenDOS] $P$G
+SET OPENDOSCFG=D:\OPENDOS
+CALL D:\EMUBIN\UXMACROS.BAT
```

Comment:
> Of course that is already there and that hides the C:\> from being shown, but DOS
> will still process it as an extra line.

I've added multiple empty lines to my autoexec and
I don't see the difference. So what you say is strange,
could you attach screenshots? Is this problem specific
to OpenDOS?